internal/host/hostutil/host_utils.go

```diff
@@ -154,6 +156,11 @@ func IgnitionFileName(host *models.Host) string {
 	return fmt.Sprintf("%s-%s.ign", common.GetEffectiveRole(host), host.ID)
 }
 
+func IsAssistedServiceHost(h *models.Host) bool {
```

Comment:
Can we get here some comment on what creates the file mentioned here and how this function is meant to be used? It will live in utils, so there is a potential that someone in the future will try to use it for another reason; if that's the case, some note that there is another entity creating the `localHostIDFile` would be helpful